### PR TITLE
Adding Group Variables support

### PR DESCRIPTION
**Pull request by pmwebster**

### CODE REVIEW

File: group_variables.go [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+	}
+
+	return vs, resp, err
+}
+
+// ShowVariableDetails gets a variable.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/group_level_variables.html#show-variable-details
+func (s *GroupVariablesService) ShowVariableDetails(gid interface{}, key string, options ...OptionFunc) (*GroupVariable, *Response, error) {
```
Comment (svanharmelen) [inline group_variables.go]: I prefer this to be called just `GetVariable`. Did guess I missed that in the project variables PR. Would be good to update that one as well...

File: group_variables.go [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+func (v GroupVariable) String() string {
+	return Stringify(v)
+}
+
+// ListGroupVariables gets a list of all variables in a group.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/group_level_variables.html#list-group-variables
+func (s *GroupVariablesService) ListGroupVariables(gid interface{}, options ...OptionFunc) ([]*GroupVariable, *Response, error) {
```
Comment (svanharmelen) [inline group_variables.go]: I prefer this to be called just `ListVariables`. Did guess I missed that in the project variables PR. Would be good to update that one as well...

**Comment by pmwebster** [comment]
Not sure why that is failing; If you look at the diff the duplicate structs are gone :/

**Comment by svanharmelen** [comment]
LGTM! Thanks ⚡️

**Pull request merged**